src/components/Input/Select.md

```diff
@@ -0,0 +1,67 @@
+### Default Style
+
+```jsx
+<Select type="text" name="text-demo" label="Select Input" note="This is a note." margin={{ bottom: 'md' }}>
+    <option>Choose an option...</option>
+    <option value="Option One">Option One</option>
+    <option value="Option One">Option Two</option>
+    <option value="Option One">Option Three</option>
+    <option value="Option One">Option Four</option>
+</Select>
+```
+
+### Select Inline Validation
+
+```jsx
+<Select type="text"
+       name="select-valid-state"
+       variant="valid"
+       label="Type"
+       note={<> You must pick a <strong>Type</strong> of meats. </>}
+       margin={{bottom: 'md'}}>
+    <option value="Steak">Choose an option...</option>
```

Comment:
Should this option not have a value (same with the other validation examples)?

Reply:
Actually, I would think it should have a value of empty string, as by default the value is the child of the option (i.e., all the `value` attributes that are the same as the child are redundant). Otherwise, any validation would have to check for the string "Choose an option..."